data/swc_environment.yml

```
@@ -0,0 +1,9 @@
+name: swc
+channels:
+  - conda-forge
+dependencies:
+  - ipykernel
+  - jupyter
+  - matplotlib
+  - numpy
+  - pandas
```

```suggestion
name: carpentries
channels:
  - conda-forge
dependencies:
  - ipykernel
  - jupyter
  - matplotlib
  - numpy
  - pandas
  - plotnine
  - plotly
  - seaborn
```

Add a few more libraries to support the DC and LC (novice) Python lessons as well.

---

This would also require the filename to change (e.g. `cp_environment.yml`)